src/main/resources/org/jenkinsci/plugins/configfiles/ConfigFilesUI/setup-codemirror.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener("DOMContentLoaded", function () {
+    const textarea = document.getElementById("config.content");
+    const contentTypeElement = document.querySelector("span[data-content-type]");
+
+    if (textarea && contentTypeElement) {
+        const contentTypeVal = contentTypeElement.getAttribute("data-content-type");
+        const isReadOnly = contentTypeElement.hasAttribute("data-read-only");
```

Comment:
Would this not erroneously set `isReadOnly` to true for `data-read-only="false"`?

Reply:
This isn't Python, but this nugget from the [Zen of Python](https://peps.python.org/pep-0020/) comes to mind:

> Explicit is better than implicit.

I vote for number 2.

Reply:
Got it. Thank you!